src/components/block/DonationProjectCard/DonationProjectCard.js

```diff
@@ -0,0 +1,27 @@
+import PropTypes from "prop-types";
+import React from "react";
+
+import s from "./DonationProjectCard.module.css";
+
+const DonationProjectCard = ({ title, subtitle, background }) => (
+  <div
+    className={s.container}
+    style={{ backgroundImage: `url('${background}')` }}
+  >
+    <div className={s.bottombox}>
+      <div className={s.bar} />
+      <div>
+        <div className={s.title}>{title}</div>
+        <div className={s.subtitle}>{subtitle}</div>
+      </div>
+    </div>
+  </div>
+);
+
+DonationProjectCard.propTypes = {
+  title: PropTypes.string.isRequired,
+  subtitle: PropTypes.string.isRequired,
```

Comment:
I think it'd be better to keep the property names that serve the same purpose to be consistent throughout the Card components. Do you mind changing this 'subtitle' key to 'description'?